src/component/actionbar/index.tsx

```diff
@@ -12,9 +12,10 @@ import { TimelineContext } from "context/TimelineContext";
 import { useCreateUpdate } from "module/hook/update";
 import { ITimeline } from "module/interface/timeline";
 import { IUser } from "module/interface/user";
+import { IVenture } from "module/interface/venture";
 
 interface ActionBarProps extends DefaultActionBarProps {
-  ventureId: string;
+  currentVenture: IVenture;
```

Comment:
Pass in the whole venture instead of just the ID so we can check which timelines the current user has access to for creating updates.